src/lib/dashboard/data.ts

```diff
@@ -0,0 +1,147 @@
+import { formatCurrency } from "$lib/helpers/format";
+import { writable } from "svelte/store";
+
+interface Budget {
+    value: string,
+    change: number,
+    changeType: 'down' | 'up'
+}
+
+export const budget = writable<Budget>({
+    value: formatCurrency(24000),
```

Comment:
please format in *.svelte file (these are just data that should not be formatted)